.github/CODEOWNERS

```diff
@@ -0,0 +1,24 @@
+# Global configs
+.github/ @abhinavrastogi-harness @knagurski
+.eslintrc.js @abhinavrastogi-harness @knagurski
+.prettierrc.yml @abhinavrastogi-harness @knagurski
+
+# Common ownership for shared config files
+/packages/*/package.json @abhinavrastogi-harness @knagurski
+/apps/*/package.json @abhinavrastogi-harness @knagurski
+/packages/*/vite.config.ts @abhinavrastogi-harness @knagurski
+/apps/*/vite.config.ts @abhinavrastogi-harness @knagurski
```

Comment:
@vardanbansal-harness you should also be included on the shared config files

Reply:
Done.